app/controllers/likes_controller.rb

```diff
@@ -6,4 +6,18 @@ def show; end
   def new; end
 
   def edit; end
+
+  def create
+    @like = Like.new
+    @like.author = current_user
+    @like.post = Post.find(params[:post_id])
+    @like.save
+    redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+  end
```

Comment:
- [ ] On the post show page there should be a button for creating a like. I cannot find this button. Kindly add it.